pkg/k8sutil/k8sutil.go

```diff
@@ -99,19 +98,19 @@ func GetPod(ctx context.Context, client crclient.Client, ns string) (*corev1.Pod
 
 	log.V(1).Info("Found podname", "Pod.Name", podName)
 
-	pod := &corev1.Pod{
```

Comment:
I removed the initial setting as it gets removed anyway, and it didn't seem to influence the .Get method in any way for me locally. 🤷‍♀️